src/components/organism/SwapTokens/index.tsx

```diff
@@ -149,39 +148,39 @@ const SwapTokens = () => {
           : selectedTokens.tokenA.decimals
       );
 
-      const assetTokenPrice = await getNativeTokenFromAssetToken(
+      const nativeTokenPrice = await getNativeTokenFromAssetToken(
         api,
         selectedTokens?.tokenA?.tokenSymbol === TokenSelection.NativeToken
           ? selectedTokens?.tokenB?.tokenId
           : selectedTokens?.tokenA.tokenId,
         valueWithDecimals
       );
 
-      if (assetTokenPrice) {
-        const assetTokenNoSemicolons = assetTokenPrice.toString().replace(/[, ]/g, "");
-        const assetTokenNoDecimals = formatDecimalsFromToken(
-          parseFloat(assetTokenNoSemicolons),
+      if (nativeTokenPrice) {
+        const nativeTokenNoSemicolons = nativeTokenPrice.toString().replace(/[, ]/g, "");
+        const nativeTokenNoDecimals = formatDecimalsFromToken(
+          parseFloat(nativeTokenNoSemicolons),
           selectedTokens?.tokenA?.tokenSymbol === TokenSelection.NativeToken
             ? selectedTokens.tokenA.decimals
             : selectedTokens.tokenB.decimals
         );
 
-        const assetTokenWithSlippage =
+        const nativeTokenWithSlippage =
           inputEdited.inputType === InputEditedType.exactIn
-            ? assetTokenNoDecimals - (assetTokenNoDecimals * slippageValue) / 100
-            : assetTokenNoDecimals + (assetTokenNoDecimals * slippageValue) / 100;
+            ? nativeTokenNoDecimals - (nativeTokenNoDecimals * slippageValue) / 100
+            : nativeTokenNoDecimals + (nativeTokenNoDecimals * slippageValue) / 100;
```

Comment:
Same here. Essentially wherever u have some kind of calculation, use decimal package